### PR TITLE
Added the Versions files in Doc

### PR DESCRIPTION
**Pull request by SeqLaz**
Added the Files version section in the Projects page, and added a step-by-step for deleting Old files version in QFieldCloud.

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2747 Document how to remove old versions of the same file](https://app.clickup.com/t/2192114/QF-2747)

**Comment by SeqLaz** [comment]
@suricactus Is this a good place for placing the Files version in the documentation?

File: documentation/reference/qfieldcloud/projects.en.md [hunk 1]
```diff
@@ -37,6 +37,24 @@ project
 └── project.qgs
 ```
 
+## Files versions
```
Comment (suricactus) [inline documentation/reference/qfieldcloud/projects.en.md:40]:
```suggestion
## File versions
```

File: documentation/reference/qfieldcloud/projects.en.md [hunk 2]
```diff
@@ -37,6 +37,24 @@ project
...
+
+Files version is a method used by QFieldCloud to restore to a previous version of the modified files. The files versions can be found in in **Files** section of your project, the **Community plan** will allow to store 3 files versions, **Pro plan** and **Organization plan** will allow to store 10 files versions.
```
Comment (suricactus) [inline documentation/reference/qfieldcloud/projects.en.md:42]:
```suggestion
QFieldCloud uses file versioning that allows you to restore to a previous version of the modified files. The files and file versions can be found in in **Files** section of your project. Each subscription plan has different default number of versions stored for each file. Check the qfield.cloud [https://qfield.cloud/pricing.html](pricing page for further details).
```

File: documentation/reference/qfieldcloud/projects.en.md [hunk 3]
```diff
@@ -37,6 +37,24 @@ project
...
+
+### How to remove old versions of the files
```
Comment (suricactus) [inline documentation/reference/qfieldcloud/projects.en.md:44]:
```suggestion
### Deleting old file versions
```

File: documentation/reference/qfieldcloud/projects.en.md [hunk 4]
```diff
@@ -37,6 +37,24 @@ project
...
+
+The saved versions, can be deleted. If there is a specific file or versions that like to remove, this can manually be delete from the project's **File** section. This can help free up space and ensure that only relevant versions files are retained.
```
Comment (suricactus) [inline documentation/reference/qfieldcloud/projects.en.md:46]:
```suggestion
To ensure only relevant file versions are retained or to reduce the amout of storage needed for your account, you can delete obsolete file versions. You can manually delete file versions from the project's **File** section.
```

**Review by suricactus — APPROVED** [review]
Some small comments but looks good to me.

@lucienicolier ?

**Comment by lucienicolier** [comment]
> Some small comments but looks good to me.
> 
> @lucienicolier ?

Perfect for me!

**Pull request merged**